app/views/layouts/_header.html.erb

```diff
@@ -93,4 +93,18 @@
   if (window.location.href.includes('search')) {
       searchBar.style.display = "block";
   }
+  <script type="text/javascript">
```

Comment:
This seems like a workaround, Can we do this without js ?

Reply:
Sir any suggestions on what other approach can be used? At the moment I can only think of this fix.

Reply:
@h2o-Noob I reproduce the error and I think the error lies in the ordering of files in https://github.com/CircuitVerse/CircuitVerse/blob/7cc19ee5f624f713005b8b262bc292c9d8a262c2/app/javascript/controllers/groups_controller.js#L1

I think this ordering show works as it is working for me.
```
import { Controller } from 'stimulus';
import * as $ from 'jquery/dist/jquery';
import 'js/jquery.js';
import 'bootstrap/dist/js/bootstrap.min.js';
import 'bootstrap/dist/js/bootstrap';
import 'bootstrap-tagsinput/dist/bootstrap-tagsinput.js';
```

WDYT @tachyons ?